src/pc/stats.rs

```diff
@@ -216,8 +216,13 @@ impl Display for ParachainStats {
 		)?;
 		writeln!(
 			f,
-			"Skipped slots: {}, slow availability: {}, slow bitfields propagation: {}",
-			self.skipped_slots.to_string().bright_purple(),
+			"Skipped slots: {}, {:?}",
+			self.skipped_slots.len().to_string().bright_purple(),
```

Comment:
I prefer to have a separate line for the hashes and keep this as is.

Reply:
Done